src/main/scala/se/nimsa/sbx/app/routing/AnonymizationRoutes.scala

```diff
@@ -52,6 +52,7 @@ trait AnonymizationRoutes {
       post {
         entity(as[Seq[TagValue]]) { tagValues =>
           val source = anonymizedDicomData(imageId, tagValues, storage)
+            .batchWeighted(storage.streamChunkSize, _.length, identity)(_ ++ _)
```

Comment:
This line allows for a faster upstream to proceed when downstream is slow. Also creates larger chunks of data so that fewer chunks need to be sent to the client.